pipeline_dp/combiners.py

```diff
@@ -460,6 +475,48 @@ def metrics_names(self) -> List[str]:
         return self._metrics_to_compute
 
 
+class VectorSumCombiner(Combiner):
+    """Combiner for computing dp vector sum.
+
+    The type of the accumulator is np.ndarray, which represents sum of the vectors of the same size
+    for which this accumulator is computed.
+    """
+    AccumulatorType = np.ndarray
+
+    def __init__(self, params: CombinerParams):
+        self._params = params
+
+    def create_accumulator(self,
+                           values: Iterable[ArrayLike]) -> AccumulatorType:
+        array_sum = None
+        for val in values:
+            if type(val) is not np.ndarray:
```

Comment:
Nit (just learned :) ): More idiomatic way: 
`if not isinstance(val, np.ndarray):`